<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StressTestApp</class>
 <widget class="QMainWindow" name="StressTestApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StressTestApp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>160</width>
      <height>164</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="cpu_checkBox">
       <property name="text">
        <string>CPU</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="fpu_checkBox">
       <property name="text">
        <string>FPU</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cache_checkBox">
       <property name="text">
        <string>Cache</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ram_checkBox">
       <property name="text">
        <string>RAM</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="localDisks_checkBox">
       <property name="text">
        <string>Local disk(s)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="gpu_checkBox">
       <property name="text">
        <string>GPU(s)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>137</x>
      <y>319</y>
      <width>551</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="start_pushButton">
       <property name="text">
        <string>Старт</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stop_pushButton">
       <property name="text">
        <string>Стоп</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clear_pushButton">
       <property name="text">
        <string>Очистить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Начался:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="currentDateTime_label">
       <property name="text">
        <string>currentDateTime</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Прошло:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="timer_label">
       <property name="text">
        <string>00:00:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
